### ISSUE TITLE
Render issues with event in nodebb-plugin-category-sections

### ISSUE THREAD

**Issue by Fank** [pitaj/nodebb-plugin-calendar#84]
We use the `nodebb-plugin-category-sections` which shows the last post under the category:
![image](https://user-images.githubusercontent.com/1900106/27450057-c313bb66-578a-11e7-8e42-84127b422562.png)

Post in thread:
![image](https://user-images.githubusercontent.com/1900106/27450068-cd725aae-578a-11e7-872e-395844552a1f.png)

**Comment by Fank** [comment]
We use the lavender theme

**Comment by pitaj** [comment]
Does this occur on all languages? Does it work with `en-US`?

**Comment by Fank** [comment]
Nope still same
![screenshot_20170624-143120](https://user-images.githubusercontent.com/1900106/27508637-df2a9972-58e9-11e7-900b-a7164bbfa97a.png)

**Comment by Fank** [comment]
Looks like its theme related because its not related to an error or something like that.

**Comment by pitaj** [comment]
Try running `./nodebb build lang`

**Comment by Fank** [comment]
New nodebb installation with calendar plugin and lavendar theme:

![image](https://user-images.githubusercontent.com/1900106/27511871-8d52223c-592f-11e7-89d8-7f8b7aea56b6.png)

**Comment by spiritax** [comment]
I dont know if it's related, but i've a similar issue :

<img width="506" alt="screen shot 2017-06-25 at 11 56 19" src="https://user-images.githubusercontent.com/10990671/27515144-897fb750-599d-11e7-9ded-83043ed04a0f.png">

**Comment by Fank** [comment]
This is caused by the "preview" handling it cuts the
```html
<div class="plugin-calendar-event panel panel-success" data-translated="false">
```
https://github.com/pitaj/nodebb-plugin-calendar/blob/735a06a831cd80cfb9a29303c2d921db21fbf4b1/src/lib/templates.js#L109

 and so the translate won't be triggered because it only searches for
```
data-translated="false"
```

https://github.com/pitaj/nodebb-plugin-calendar/blob/735a06a831cd80cfb9a29303c2d921db21fbf4b1/src/client/clientSideTranslation.js#L5

I don't have a idea how to fix this.

Example:
![image](https://user-images.githubusercontent.com/1900106/27661836-1edeec42-5c5c-11e7-92ff-3ee4d088ed9c.png)
![image](https://user-images.githubusercontent.com/1900106/27661843-270cee1e-5c5c-11e7-8875-ff0e36c94062.png)

**Comment by pitaj** [comment]
Are you sure `data-translated="false"` doesn't exist in the HTML output by the server? You should be able to check with" view source" in your browser.

**Comment by Fank** [comment]
I did its the same as in the pic i posted above.

**Comment by pitaj** [comment]
Ok I can confirm this. Will see if I can find a fix.

**Comment by theportablegeek** [comment]
Has anyone found a fix for this yet?

**Comment by pitaj** [comment]
Pretty sure I can fix this. I just haven't got around to it yet

**Comment by pitaj** [comment]
Pretty sure I can fix this. I just haven't got around to it yet. I realize it's been a while, but I'll check it out in the coming week.

**Comment by pitaj** [comment]
Okay, I've looked into it.

- It isn't nodebb-plugin-category-sections, it's just the lavender theme
- It isn't anything my plugin is doing
- NodeBB [strips tags from recent replies](https://github.com/NodeBB/NodeBB/blob/master/src/categories/recentreplies.js#L27)
- There is a way around it with just my plugin, but instead I'm going to try to make a change to core so I can avoid the headache of a terrible hack (move [this hook](https://github.com/NodeBB/NodeBB/blob/master/src/posts/summary.js#L84) to before the stripTags runs)

**Comment by theportablegeek** [comment]
What would be the solution with just your plugin? Would it work with the persona theme as well?

**Comment by pitaj** [comment]
It would be wrapping the event in an element that won't get sanitized away, and then replacing it with a button in that hook if it doesn't contain the div event element. But it's a lot easier to just get the hook moved, and a lot less of a pain for later.

Either way, the fix should work regardless of theme.

**Comment by pitaj** [comment]
Fixed in 83e1de7f34e6237d81671da32b628e33c5d676c6
